conf/setup.yaml

```diff
@@ -45,3 +45,11 @@ ha:
     ha-cleanup:
         script: /opt/seagate/cortx/ha/conf/script/build-cortx-ha
         args: cleanup
+
```

Comment:
Why are these sections added as part of this PR?

Reply:
add_node , remove_node were earlier part of conf/iostack-ha/setup.yaml
Ajay P mentioned that they were mistakenly added there instead of adding to conf/setup.yaml ;
and requested the change.

Proivsioner also uses these calls ; so due to the dependency they are added to this PR 

Please find the relevant discussion/comment from the jira here :
https://jts.seagate.com/browse/EOS-12885?focusedCommentId=1946429&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1946429